docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -42,8 +63,254 @@ then try adding ``--ssh-key-file ~/.ssh/<name>``
+  * You have the option to use vanilla NGINX without HTTPS support or an
+    OpenResty NGINX integrated with 3scale API Gateway.
+
+
+Step 4.1: Vanilla NGINX
+^^^^^^^^^^^^^^^^^^^^^^^
+
+   * This configuration is located in the file ``nginx/nginx-svc.yaml``.
+    
+   * Set the ``metadata.name`` and ``metadata.labels.name`` to the value
+     set in ``ngx-instance-name`` in the ConfigMap above.
+   
+   * Set the ``spec.selector.app`` to the value set in ``ngx-instance-name`` in
+     the ConfigMap followed by ``-dep``. For example, if the value set in the
+     ``ngx-instance-name`` is ``ngx-instance-0``, set  the
+     ``spec.selector.app`` to ``ngx-instance-0-dep``.
+   
+   * Start the Kubernetes Service:
+
+     .. code:: bash
+     
+        $ kubectl --context k8s-bdb-test-cluster-0 apply -f nginx/nginx-svc.yaml
+
+
+Step 4.2: OpenResty NGINX + 3scale
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   
+   * This configuration is located in the file ``nginx/nginx-3scale-svc.yaml``.
+
+   * You have to enable HTTPS for this one and will need an HTTPS certificate
+     for your domain
+      
+   * You should have already created the Kubernetes Secret in the previous
+     step.
+
+   * Set the ``spec.selector.app`` to the value set in ``ngx-instance-name`` in
+     the ConfigMap followed by ``-dep``. For example, if the value set in the
+     ``ngx-instance-name`` is ``ngx-instance-0``, set  the
+     ``spec.selector.app`` to ``ngx-instance-0-dep``.
+   
+   * Start the Kubernetes Service:
+   
+     .. code:: bash
+
+        $ kubectl --context k8s-bdb-test-cluster-0 apply -f nginx-3scale/nginx-3scale-svc.yaml
+
+
+Step 5: Assign DNS Name to the NGINX Public IP
+----------------------------------------------
+
+  * This step is required only if you are planning to set up multiple
+    `BigchainDB nodes
+    <https://docs.bigchaindb.com/en/latest/terminology.html>`_ or are using
+    HTTPS certificates tied to a domain.
+
+  * The following command can help you find out if the nginx service started
+    above has been assigned a public IP or external IP address:
+   
+    .. code:: bash
+
+       $ kubectl --context k8s-bdb-test-cluster-0 get svc -w
+   
+  * Once a public IP is assigned, you can log in to the Azure portal and map it to
+    a DNS name.
+   
+  * We usually assign ``bdb-test-cluster-0``, ``bdb-test-cluster-0`` and
+    so on in our documentation.
+   
+  * Let us assume that we assigned the unique name of ``bdb-test-cluster-0`` here.
+
+
+**Set up DNS mapping in Azure.**
+
+Select the current Azure resource group and look for the ``Public IP``
+resource. You should see at least 2 entries there - one for the Kubernetes
+master and the other for the MongoDB instance. You may have to ``Refresh`` the
+Azure web page listing the resources in a resource group for the latest
+changes to be reflected.
+Select the ``Public IP`` resource that is attached to your service (it should
+have the Kubernetes cluster name along with a random string),
```

Comment:
Both of the `Public IP` resources that I see in the Azure Portal have the "cluster name" (actually, not the cluster name, the "DNS prefix" I gave in the original `az acs create` command). And both are followed by a random-looking string. So I'm stuck. Only one of the `Public IP` resources contains `k8s-master` in its name. Should I pick that one? Or the other one?